testing/thunderbird/APKBUILD

```diff
@@ -0,0 +1,296 @@
+# Contributor: Joseph Benden <joe@benden.us>
+# Maintainer: Joseph Benden <joe@benden.us>
+
+pkgname=thunderbird
+pkgver=60.8.0
+_pkgver=$pkgver
+_xulver=$pkgver
+pkgrel=0
+pkgdesc="Thunderbird email client"
+url="https://www.thunderbird.net/"
+arch="x86_64"
+license="GPL-3.0-or-later AND LGPL-2.1-or-later AND MPL-2.0"
+options="!check !strip"
+depends="
+	alsa-lib
+	dbus-glib
+	ffmpeg
+	gconf
+	gtk+3.0
+	hunspell
+	icu
+	libevent
+	libjpeg-turbo
+	libnotify
+	libogg
+	libtheora
+	libvorbis
+	libxcomposite
+	libxt
+	nspr
+	nss
+	sqlite
+	startup-notification
+	"
+makedepends="
+	alsa-lib-dev
+	autoconf2.13
+	automake
+	bsd-compat-headers
+	bzip2-dev
+	cargo
+	cbindgen
+	clang-dev
+	dbus-glib-dev
+	ffmpeg-dev
+	gconf-dev
+	gtk+2.0-dev
+	gtk+3.0-dev
+	hunspell-dev
+	icu-dev
+	libevent-dev
+	libidl-dev
+	libjpeg-turbo-dev
+	libnotify-dev
+	libogg-dev
+	libtheora-dev
+	libtool
+	libvorbis-dev
+	libvpx-dev
+	libxcomposite-dev
+	libxt-dev
+	llvm-dev
+	mesa-dev
+	nasm
+	nodejs
+	nspr-dev
+	nss-dev
+	nss-static
+	paxmark
+	python3-dev
+	sed
+	sqlite-dev
+	startup-notification-dev
+	wireless-tools-dev
+	yasm
+	zip
+	"
+source="https://ftp.mozilla.org/pub/thunderbird/releases/$pkgver/source/thunderbird-$pkgver.source.tar.xz
+	stab.h
+	fix-fortify-system-wrappers.patch
+	fix-fortify-inline.patch
+	fix-seccomp-bpf.patch
+	fix-toolkit.patch
+	fix-tools.patch
+	mallinfo.patch
+	mozilla-build-arm.patch
+	disable-moz-stackwalk.patch
+	fix-rust-target.patch
+	fix-bug-1261392.patch
+	fix-webrtc-glibcisms.patch
+	sandbox-membarrier.patch
+	rust-unitialized-field.patch
+	rust133.patch
+	0003-Disable-ccache-stats.patch
+	thunderbird.desktop
+	"
+_mozappdir=/usr/lib/$pkgname
+
+# Google API keys (see http://www.chromium.org/developers/how-tos/api-keys)
+# Note: These are for Arch Linux use ONLY. For your own distribution, please
+# get your own set of keys. Feel free to contact foutrelis@archlinux.org for
+# more information.
+_google_api_key=AIzaSyDwr302FpOSkGRpLlUpPThNTDPbXcIn_FM
```

Comment:
> These are for Arch Linux use ONLY.

Reply:
Yes. Upon Alpine accepting this new port, they will need to acquire their own API keys.

I did in fact mention this in bold, within the summary of the PR.

Thanks!
-Joe